pom.xml

```diff
@@ -47,89 +62,424 @@
 		</developer>
 	</developers>
 
-	<pluginRepositories>
-		<pluginRepository>
-			<id>repo.jenkins-ci.org</id>
-			<url>http://repo.jenkins-ci.org/public/</url>
-		</pluginRepository>
-	</pluginRepositories>
+	<properties>
+		<!-- Plugin dependencies -->
+		<jenkinsCoreVersion>1.651.1</jenkinsCoreVersion>
+		<gitPluginVersion>2.4.4</gitPluginVersion>
+		<jqueryPluginVersion>1.11.2-0</jqueryPluginVersion>
+		<jodaTimeVersion>2.9.2</jodaTimeVersion>
+		<commonsLang3Version>3.4</commonsLang3Version>
+		<!-- Jenkins dependencies -->
```

Comment:
I don't agree for this change. Maven is a tool, which resolve dependency. I don't need set this explicite. Any plugin which I know, has not added such dependence.
Eg.
https://github.com/jenkinsci/git-plugin/blob/master/pom.xml
https://github.com/jenkinsci/git-client-plugin/blob/master/pom.xml
https://github.com/jenkinsci/active-choices-plugin/blob/master/pom.xml
https://github.com/jenkinsci/scm-api-plugin/blob/master/pom.xml
and many, many more